das/das.go

```diff
@@ -97,6 +97,8 @@ func Serialize(c arbstate.DataAvailabilityCertificate) []byte {
 
 	buf = append(buf, arbstate.DASMessageHeaderFlag)
 
+	buf = append(buf, c.KeysetHash[:]...)
```

Comment:
Should KeysetHash also be one of the fields that gets signed (`serializeSignableFields`)?

Reply:
I think not, because the signer might have a one-DAS keyset, but the aggregator might use the signature in a committee keyset.